packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -111,14 +115,13 @@ class FluentLocalizer {
       // to pass down to fluent
       const arr = plainTextArr[i].match(/([a-zA-z-]*\s=\s"([^"]+)")/g) || '';
       if (arr[0]) {
-        let [key, val] = arr[0].split('=');
-        key = key.replace(/\s/g, '');
-        val = val.replace(/"/g, '').trim();
+        let [key, val] = [
```

Comment:
I had to modify this logic a bit since it was failing when the value had a `=` in between. Failing url: `https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account?utm_medium=email&utm_campaign=fx-low-recovery-codes&utm_content=fx-support⁩`